Update release versions (#661)
ember-learn/ember-website: data/project/ember/beta.md

  - /ember\./
  - /ember-template-compiler/
repo: emberjs/ember.js
initialVersion: 3.20.0 # Manually update, see https://libraries.io/npm/ember-source throughout

> lastRelease: 3.21.0-beta.2 # Manually update
> futureVersion: 3.21.0-beta.3 # Manually update

Reviewer: It looks like `3.21.0-beta.3` has been released. Would these be instead:
```suggestion
lastRelease: 3.21.0-beta.3 # Manually update
futureVersion: 3.21.0-beta.4 # Manually update
```

finalVersion: 3.21.0 # Manually update
channel: beta

> cycleEstimatedFinishDate: 2020-09-09 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release

Reviewer 2: ```suggestion
cycleEstimatedFinishDate: 2020-08-24 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
```

> cycleEstimatedFinishDate: 2020-08-24 12:00:00 GMT # Manually update, change next one to 6 weeks from this date...regardless of delays in the release

Reviewer 2: Preview is showing "Invalid date" due to time being included.

```suggestion
cycleEstimatedFinishDate: 2020-08-24 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
```

date: 2020-07-27 # Manually update, get date for `lastRelease`

> nextDate: 2020-09-09 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release

Reviewer 2: ```suggestion
nextDate: 2020-08-24 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
```

> nextDate: 2020-08-24 12:00:00 GMT # Manually update, change next one to 6 weeks from this date...regardless of delays in the release

Reviewer 2: ```suggestion
nextDate: 2020-08-24 # Manually update, change next one to 6 weeks from this date...regardless of delays in the release
```

Reviewer 3: @locks that pinned time was needed because tests failed for me in the eastern us time zone.

Reviewer: @locks Where did you see "Invalid date" in the preview? I guessed you were referring to the Releases page, but saw August 24.

<img width="800" alt="Screen Shot 2020-07-31 at 7 18 10 PM" src="https://user-images.githubusercontent.com/16869656/89089404-b11dd700-d362-11ea-96b2-6509ce96f3e0.png">

@jaredgalanis Would it help pass the test in your time zone if, instead of checking for exactness, we allow a tolerance? Something like,
```javascript
const tolerance = 24 * 60 * 60 * 1000; // 1 day in milliseconds

assert.ok(
  Math.abs(data.cycleEstimatedFinishDate - nextExpectedRelease) < tolerance,
  `the next expected release should be on ${nextExpectedRelease} but we have set ${data.cycleEstimatedFinishDate}`
);
```

Reviewer 2: @ijlee2 Yes, I see "Invalid DateTime" instead of the expected dates:

<img width="991" alt="image" src="https://user-images.githubusercontent.com/32344/89104373-ec76df00-d410-11ea-9016-98261e2c700d.png">

Reviewer 3: Thanks for catching this @locks. It seems like the issue was Firefox's inability to handle a timezone string. Removing "GMT" from the date time string seems to have resolved the issue.
